### ISSUE TITLE
Can regression models produce a better output

### ISSUE THREAD

**Issue by alexgcsa** [eubr-bigsea/citrus#101]
This is just to standardize with the classification models.

Currently, the output of a regressor is just the metric RMSE:

![image](https://user-images.githubusercontent.com/24958541/54633862-c7720e00-4a5f-11e9-8a9f-f58783407c10.png)

Can't it be produced an evaluation table as it happen when I execute some classifier?

**Comment by alexgcsa** [comment]
I believe it would better to output all metrics not just one as the user chooses.

This is currently happening on the classification part.

**Comment by alexgcsa** [comment]
For instance, generalized linear regression outputs a more informative output:


![image](https://user-images.githubusercontent.com/24958541/54640793-66eacd00-4a6f-11e9-8b2b-23debd1e4eba.png)

**Comment by waltersf** [comment]
Not all models produced by Spark have extra information. Please, review the available information present in http://spark.apache.org/docs/2.4.0/api/python/pyspark.ml.html. Focus on model (e.g. RandomForestModel) and Summary (e.g. GeneralizedLinearRegressionSummary).

**Comment by alexgcsa** [comment]
Shouldn't we standardize?

I mean, we should include the same information for all classifiers in my perspective. Perhaps, to produce a table with all common regression measures (as it happens when we run a classifier).

In my perspective, it is strange to produce different outputs to algorithms in the same category. But I may be wrong about it.

**Comment by waltersf** [comment]
No, we should not. Otherwise, the common denominator would be removing all visualizations because not all regressions provide enough data (at least in Spark) to plot graphs. And we already have a boolean flag for this. It is the option "Exibir imagens e gráficos (se disponíveis)".

**Comment by alexgcsa** [comment]
Alright, I forgot about this option "Exibir imagens e gráficos (se disponíveis)".

I am going to check on Spark or Scikit-learn about these plots. I didn't have enough time to do it.

Thanks,

Alex
